go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cosmos/go-bip39 v0.0.0-20180618194314-52158e4697b8
 	github.com/cosmos/ledger-cosmos-go v0.10.3
 	github.com/fortytw2/leaktest v1.3.0 // indirect
+	github.com/go-kit/kit v0.8.0
```

Comment:
Why are we adding a new dep?

Reply:
not sure how that got in there, removing